freqtrade/exchange/exchange.py

```diff
@@ -23,6 +23,7 @@
                                   OperationalException, TemporaryError)
 from freqtrade.exchange.common import BAD_EXCHANGES, retrier, retrier_async
 from freqtrade.misc import deep_merge_dicts, safe_value_fallback
+from freqtrade.pairlist.pairlistmanager import ListPairsWithTimeframes
```

Comment:
This import causes a circular import (exchange needs `freqtrade.pairlist` - and `freqtrade.pairlist` needs exchange.

I guess if we intend to use these type-stubs more frequently, the best place might be `freqtrade.typing` ... so circular dependencies are not an issue.

Reply:
that's what I hate python for...

Reply:
why?
every other language i ever used will kill you for circular dependencies too - so why hate python for it ... :D

Reply:
it does not report circular dependencies (even static checkers we use) and always print errors that are not relevant to the problem

in C, we always surrounded headers with
```
#ifndef HEADERFILE_H
#define HEADERFILE_H
```
"standard" construction and this was the perfect way to live with this and to make the headers really *idempotent*... At least, there was a recipe to live with this... no such a well-known recipe for python...